docs/installation/bootstrap.rst

```diff
@@ -52,8 +50,9 @@ system is configured to re-mount them automatically after a reboot.
 
 .. _Bootstrap SSH Provisioning:
 
-SSH provisioning
-^^^^^^^^^^^^^^^^
+SSH Provisioning
```

Comment:
One comment about the format of the commit message: we don't put reference to the issue in the title line (we usually use the footer for that).

See [contributions guideline](https://github.com/scality/metalk8s/blob/development/2.4/CONTRIBUTING.md#committing-guidelines) (and/or the [developer doc](https://metal-k8s.readthedocs.io/en/latest/developer/best_practices/commit.html#how-to-write-a-commit-message) for more information).

e.g.:

```
doc: improve wording + minor fix

Improve the documentation to make it customer-facing.

Refs: #1995
Refs: #2112
```